sandy/core/cov.py

```diff
@@ -540,18 +548,18 @@ def sampling(self, nsmp, seed=None, rows=None, pdf='normal',
 
         Examples
         --------
-        Draw 3 sets of samples using custom seed.
+        Draw 3 sets of samples using custom seed:
         >>> sandy.CategoryCov([[1, 0.4],[0.4, 1]]).sampling(3, seed=11)
                      0            1
-        0 -1.74945e+00 -3.13159e+00
-        1  2.86073e-01  1.06836e-01
-        2  4.84565e-01 -9.91209e-02
+        0 -7.49455e-01 -2.13159e+00

```

Comment:
why are the samples different? it doesn't seem to me there is any substantial change to this algorithm